tests/generation/test_utils.py

```diff
@@ -1241,6 +1241,9 @@ def test_dola_decoding_sample(self):
 
             if model.get_output_embeddings() is None:
                 self.skipTest("DoLa is not supported for models that don't have output embeddings")
+
+            logits_processor_kwargs = self._get_logits_processor_kwargs(do_sample=True, config=model.config)
```

Comment:
I'm assuming this is needed for VLMs 😅 (because it fetched special VLM kwargs)

Reply:
yes